### PR TITLE
closes #162

### LINKED ISSUE #162: menu / dash

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

File: client/js/components/menu.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var React = require("react");
+var $ = require("jquery");
+
+var Menu = React.createClass({
+  componentDidMount: function() {
+    $(document).ready(function() {
+      $(".menu").click(function () {
+        $("#dropdown").slideToggle(300);
```
Comment (dduugg) [inline client/js/components/menu.jsx]: super minor, but the [magic number](http://dduugg.github.io/naming-and-defining-constants/) should be extracted.

**Comment by dduugg** [comment]
:+1:

**Comment by cmperez** [comment]
magic number extracted

**Comment by owen-d** [comment]
looks good

**Comment by cmperez** [comment]
:gun: :gun:

**Pull request merged**